e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -0,0 +1,370 @@
+const admin_token = 'ZNhnqJyDqgkXVk_bpobS3YhIMz0='
+const admin_token_obj = {
+  id: `test-token-${Cypress.moment().format('MMDDYYhhmm')}`,
+  name: "cypress-api-test-admin-token",
+  value: admin_token
+}
+
+const avengers_project = {
+  id: `avengers-project-${Cypress.moment().format('MMDDYYhhmm')}`,
+  name: "Test Avengers Project"
+}
+
+const xmen_project = {
+  id: `xmen-project-${Cypress.moment().format('MMDDYYhhmm')}`,
+  name: "Test X-men Project"
+}
+
+let avengersRule = {
```

Comment:
does this work as a `const`? I know we're using it as part of the UPDATE but does that actually modify the variable?

Reply:
yeah we made this `let` so we could modify `avengersRule` itself. we could make this variable a const and just create a new `updatedAvengersRule`. that might be a bit cleaner